### PR TITLE
rmw 0.9.1

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rmw.rb
Comment (andy5995) [inline Formula/rmw.rb:2]:
```suggestion
  desc "Trashcan/recycle bin utility for the command-line"
```

File: Formula/rmw.rb
Comment (andy5995) [inline Formula/rmw.rb:3]:
```suggestion
  homepage "https://theimpossibleastronaut.github.io/rmw-website/"
```

**Review by andy5995 — COMMENTED** [review]
The automatic changes LGTM! I'm the rmw maintainer. These two changes I'm suggesting I can do in a separate PR, if you prefer, after this is merged as-is.

File: Formula/rmw.rb
Comment (chenrui333) [reply Formula/rmw.rb]: 👍

File: Formula/rmw.rb
Comment (chenrui333) [reply Formula/rmw.rb]: 👍

File: Formula/rmw.rb
Comment (alebcay) [inline Formula/rmw.rb:2]:
```suggestion
  desc "Trashcan/recycle bin utility for the command-line"
```

**Review by p-linnane — APPROVED** [review]

**Pull request merged**
